1.15/23.3.1/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# vim:set ft=dockerfile:
+
+# Do not edit individual Dockerfiles manually. Instead, please make changes to the Dockerfile.template, which will be used by the build script to generate Dockerfiles.
+
+# By policy, the base image tag should be a quarterly tag unless there's a
+# specific reason to use a different one. This means January, April, July, or
+# October.
+
+# Erlang version 23.3 does not support OpenSSL3, which means only for this specific version do we need to use 20.04. Please adjust accordingly
+
+FROM cimg/base:2023.04-20.04
+
+LABEL maintainer="Community & Partner Engineering Team <community-partner@circleci.com>"
+
+# Install Erlang via Erlang Solutions' .deb
+ENV ERLANG_VERSION="23.3.1"
```

Comment:
i would check to see if there are newer erlang versions, and whether they are compatible

Reply:
fixed :+1: